ses

```diff
@@ -46,133 +46,171 @@ section_header "Ceph cluster status"
     fi
-
     # timeout status commands after 5 seconds
     CT=5
-    plugin_command "ceph ${ID} --connect-timeout=$CT versions" > $LOG/ceph/ceph-versions 2>&1
-    plugin_command "ceph ${ID} --connect-timeout=$CT -s" > $LOG/ceph/ceph-status 2>&1
-    plugin_command "ceph ${ID} --connect-timeout=$CT health detail" > $LOG/ceph/ceph-health-detail 2>&1
-    plugin_command "ceph ${ID} --connect-timeout=$CT config dump" > $LOG/ceph/ceph-config-dump 2>&1
-    plugin_command "ceph ${ID} --connect-timeout=$CT mon dump" > $LOG/ceph/ceph-mon-dump 2>&1
-    plugin_command "ceph ${ID} --connect-timeout=$CT mgr dump" > $LOG/ceph/ceph-mgr-dump 2>&1
-    plugin_command "ceph ${ID} --connect-timeout=$CT osd tree" > $LOG/ceph/ceph-osd-tree 2>&1
-    plugin_command "ceph ${ID} --connect-timeout=$CT osd df tree" > $LOG/ceph/ceph-osd-df-tree 2>&1
-    plugin_command "ceph ${ID} --connect-timeout=$CT osd dump" > $LOG/ceph/ceph-osd-dump 2>&1
-    plugin_command "ceph ${ID} --connect-timeout=$CT osd pool autoscale-status" > $LOG/ceph/ceph-osd-pool-autoscale-status 2>&1
-    plugin_command "ceph ${ID} --connect-timeout=$CT df detail" > $LOG/ceph/ceph-df-detail 2>&1
-    plugin_command "ceph ${ID} --connect-timeout=$CT fs dump -f json-pretty" > $LOG/ceph/ceph-fs-dump.json 2>&1
-    plugin_command "ceph ${ID} --connect-timeout=$CT fs dump" > $LOG/ceph/ceph-fs-dump 2>&1
-    plugin_command "ceph ${ID} --connect-timeout=$CT pg dump -f json-pretty" > $LOG/ceph/ceph-pg-dump.json 2>&1
-    plugin_command "ceph ${ID} --connect-timeout=$CT pg dump" > $LOG/ceph/ceph-pg-dump 2>&1
-    plugin_command "ceph ${ID} --connect-timeout=$CT auth list" 2>&1 |
-        grep -v 'installed auth entries' |
-        sed "s/\(key:\) .*/\1 $CENSORED/g" > $LOG/ceph/ceph-auth-list
-    # `ceph report` does actually include the above information, but
-    # in JSON format.  Since adding `ceph report`, the above commands
-    # remain, because their output is easier to read in a hurry ;)
-    plugin_command "ceph ${ID} --connect-timeout=$CT report" > $LOG/ceph/ceph-report 2>&1
-    plugin_command "timeout $CT rados ${ID} df" > $LOG/ceph/rados-df 2>&1
-    plugin_command "ceph ${ID} --connect-timeout=$CT telemetry status" > $LOG/ceph/ceph-telemetry-status 2>&1
-    plugin_command "ceph ${ID} --connect-timeout=$CT balancer status" > $LOG/ceph/ceph-balancer-status 2>&1
-
-    timeout $CT ceph ${ID} osd pool ls detail 2>/dev/null |
-    sed -nEe "s/^.*'([^']+)'.* application rbd/\\1/p" |
-    while read pool; do
-        mkdir -p $LOG/ceph/images/$pool
-        timeout $CT rbd ${ID} ls $pool 2>/dev/null | tee $LOG/ceph/images/rbd-ls-${pool} 2>&1 |
-	head -n ${OPTION_SES_RBD_INFO_MAX:=10} |
-        while read image; do
-            plugin_command "timeout $CT rbd ${ID} -p $pool info $image" \
-                > $LOG/ceph/images/${pool}/rbd-info-${image} 2>&1
+
+    plugin_command "${CEPH_SHELL} ceph ${ID} --connect-timeout=$CT -s" > $LOG/ceph/ceph-status 2>&1
```

Comment:
What happens here (and below) if `$CEPH_SHELL` is undefined? Its only set if cephadm is present in [line 49](https://github.com/SUSE/supportutils-plugin-ses/pull/33/files#diff-f2b32bda85a5a4a613eb47fb01c57ce3R49).

Reply:
It'll just invoke `ceph -s`. If that fails, the subsequent `if` will skip everything else.

Reply:
OH duh...of course. Caffeine is kicking in now.